repos/fdbt-site/src/utils/index.ts

```diff
@@ -258,6 +259,25 @@ export const objectKeyMatchesExportNameExactly = (objectKey: string, exportName:
     return exportNamePart === exportName;
 };
 
-export const fareTypeIsAllowedToAddACap = (fareType: string): boolean => {
+export const fareTypeIsAllowedToAddACap = (fareType: string | undefined): boolean => {
     return !!fareType && ['single', 'return', 'flatFare'].includes(fareType);
 };
+
+export const getUniqueServices = (services: MyFaresService[]): MyFaresService[] => {
+    const uniqueServiceList: string[] = [];
+    const uniqueServices: MyFaresService[] = [];
+
+    if (!!services && services.length > 0) {
```

Comment:
If the function's arguments says it accepts an array of services as an input, we dont need !!services - we know it will be defines. We also dont need to check the length, because if we use a map function, it will return an empty array if it begins with an empty array.

Reply:
ok